### ISSUE TITLE
 Common Types of Neural Network Layers and when to use them 

### ISSUE THREAD

**Issue by WatershedXiaolan** [WatershedXiaolan/Xiaolan-s-Gitbook#215]
### Fully Connected Layer
- Used for classifying 
- Hyperparameters: Activation function; number of neurons; dropout 

### Convolution Layer
- Used for detecting features 
- Hyperparameters: dimensionality; patch; stride; number of feature map to generate; activate function; dropout; pooling

### Deconvolution Layer
- Used for Upsampling images

### Recurrent Layer
- sequential data 
- Dimensionality; structure of RNN layer; return sequence; dropout
